app/pages/Settings/Settings.js

```diff
@@ -173,17 +177,29 @@ class Settings extends Component<Props, State> {
   }
 
   renderWipe() {
-    return (
-      <Flexbox className="row">
-        <Flexbox flexDirection="column" className="description">
-          <h2 className="description-title">Wipe blockchain</h2>
-          <p>Wipe your local copy of the blockchain, with an option to wipe your wallet as well</p>
+    return this.props.walletModeStore.isFullNode() ?
```

Comment:
I prefer to break it to "if/return", to be more aligned with the rest of the codebase:

```
if (this.props.walletModeStore.isFullNode()) {
  return (...)
}
return (...)
```

Thoughts?